webapp/pages/datasubscriber/onboarding/not-connected.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, Center, VStack } from "@chakra-ui/react";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
+import { useAccount } from "wagmi";
+
+import OnBoardingCard from "~~/components/OnBoardingCard";
+import OnBoardingContentPiece from "~~/components/OnBoardingContentPiece";
+import OnBoardingHeaderComponent from "~~/components/OnBoardingHeaderComponent";
+
+const OnboardingNotConnected = () => {
+  const { isConnected } = useAccount();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (isConnected) {
+      void router.push("/user/dashboard");
+    }
+  }, [isConnected, router]);
+
+  return (
+    <>
+      <Head>
+        <title>SPN DAO - Onboarding</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Center
+        sx={{
+          flex: 1,
+        }}
+      >
+        <OnBoardingCard
+          width={"90%"}
+          alignSelf={"center"}
+          prependedContent={
+            <VStack spacing={6}>
+              <OnBoardingHeaderComponent
+                title="DALN"
+                description="Harvest data insights & stay compliant"
+              />
+              <Button colorScheme="blue" size="md">
+                Comming soon
+              </Button>
+            </VStack>
+          }
+        >
+          <Box flex="1">
+            <OnBoardingContentPiece
+              textAlign="start"
+              title="Control your data"
+              content="Have true ownership and governance in the data economy"
+              mb={10}
+            />
+            <OnBoardingContentPiece
+              textAlign="start"
+              title="Get rewards"
+              content="Get rewards in Matic whenever your data is decrypted"
+              mb={10}
+            />
+            <OnBoardingContentPiece
+              textAlign="start"
+              title="Preserve privacy"
+              content="Pool your anonymized transaction data with other DAO members"
+            />
+          </Box>
+          <Box flex="1">
+            {/* TODO: Add svg component. KEVIN Which is the best way to do this in REACT? */}
```

Comment:
@joartola  If it's an image we can use nextjs Image component with the svg as the src. If it's an icon you can copy and paste the svg code into a new component like this:

```
const SvgIcon = () => (<svg 
  xmlns="http://www.w3.org/2000/svg"
  className="h-6 w-6"
  fill="none"
  viewBox="0 0 24 24"
  stroke="currentColor"
  strokeWidth={2}
>
  <path 
  strokeLinecap="round"
  strokeLinejoin="round"
  d="M11 19l-7-7 7-7m8 14l-7-7 7-7"
/>
</svg>)
```

The advantage of using the 2nd option is that you could pass props like strokeWidth, color, etc and customize the svg.